doc/dsdr/0010-placement-modelling.md

```diff
@@ -0,0 +1,105 @@
+
+The generated candidates are composed of 5 different record types:
+H records: historically closed cases. We sample probable birthdays for each of these using the birth month as a key constraint, but also the date of entry (so joiners can't be less than zero) and the leave date (so that leavers can't be over 18).
+O records: historically open cases. We sample probable birthdays for these as above but using the extract date instead of leave date (since none yet exists).
+C records: candidate periods. These are the open periods with resampled birthdays, and we create multiple birthdays for each candidate by resampling n times, specified by the `:candidate-variations` config variable.
+P records consisting of: a) C records which are completed according to the baseline period completer algorithm (described below).
+  b) C records which are completed according to the 'age out' period completer algorithm.
+S records consisting of: a) C records which are truncated to an arbitrary point in their past and then completed with the baseline period completer algorithm.
+  b) C records which are truncated to an arbitrary point in their past and then completed with the 'age out' period completer algorithm.
+
+**The period completer algorithm**
+
+The period completer algorithm is the core process by which novel periods are generated. Its job is twofold: to decompose periods from the historic data into segments which can be reassembled like building blocks to form new periods, and thereafter to use those building blocks to complete open cases. This process is followed regardless of whether we're seeking to close an open case (P records) or generate an imagined new case (S records). S records are the result of truncating a closed case at some point in its history and then imagining a way that it might have been completed instead. The entrypoint to this algorithm is `cic.model/markov-placements-model`.
+
+*Splitting periods into segments*
+
+Each period is split into consecutive segments of 6 years. For closed cases, the partial segment at the end is retained and marked as a terminal segment. For open cases, the final segment at the end is discarded. If segments start occur within the training range (specified as n years before projection start), they're marked as `:in-filter?`. This capability allows for the possibility of filtering candidate segments to within a particular date range, although this functionality is not currently used.
+
+![Segmentation](https://raw.githubusercontent.com/MastodonC/witan.cic/dsdrs/doc/img/segmentation.png)
```

Comment:
I *think* you can do these as relative links

Reply:
So you can! 👍